.github/workflows/build.yml

```diff
@@ -0,0 +1,77 @@
+name: build_and_test
+
+on: [push, pull_request]
+
+env:
+  BUILD: 9999
```

Comment:
Why set build to 9999 instead of leaving it default? By default the build will be set to 0, which Sandstorm's updater logic understands to be a dev build. With build number 9999, the updater logic will believe this is legitimately a far-future build and this may lead to confusion if someone actually installs it.